builders/integrations/analyze/dapplooker.md

```diff
@@ -0,0 +1,83 @@
+
+This guide will cover all of the details needed to register your project with the Dapplooker platform and how to build analytics using SQL editor. This guide can be adapted for use on Moonbeam or Moonriver.
+
+
+## Steps to Connect Smart Contracts {: #steps-to-connect-smart-contract}
+
+- Signup or Login to [https://dapplooker.com/login](https://dapplooker.com/login)
+
+  ![Login to Dapplooker](/images/builders/integrations/analyze/dapplooker/dapplooker-1.png)
+
+- After you signup or log in, click on "**My Project**" button in the header section.
+
+- Click "Connect Dapp". Choose "**Connect Smart Contract**" option.
+
+  ![Connect dapp](/images/builders/integrations/analyze/dapplooker/dapplooker-2.png)
+
+- Fill in the details about your Dapp as shown below ✍️.
+- Select network as "**Moonriver" or** "**Moonbeam**".
+- Search and select smart contracts from the autocomplete.
+- Enter your website.
+- Click "**Register**" and data syncing will start. It can take some time for complete data to be synced.
+
+  ![Register your dapp](/images/builders/integrations/analyze/dapplooker/dapplooker-3.png)
+
+## Steps to Connect Subgraph {: #steps-to-connect-subgraph}
+
+- Signup/Login to [https://dapplooker.com/login](https://dapplooker.com/login)
+
+  ![Login to Dapplooker](/images/builders/integrations/analyze/dapplooker/dapplooker-1.png)
+
+- After you signup or log in, click on my project button in the header section
+
+- Click "Connect Dapp". Choose "Connect Subgraph" option
+
+  ![Connect dapp](/images/builders/integrations/analyze/dapplooker/dapplooker-4.png)
+
+- Fill in details about your Dapp as mentioned below ✍️.
+- Select network as "**Moonriver" or** "**Moonbeam**"
+- Input your Dapp subgraph endpoint.
+- Enter your website.
+- Click "Register" and data syncing will start. It can take some time for complete data to be synced.
+
+  ![Register your dapp](/images/builders/integrations/analyze/dapplooker/dapplooker-5.png)
+
+
+Once project is registered, data syncing starts.
+
+## Create Charts & Dashboard {: #create-dashboard }
+
+- After registration is done, user gets notification after data syncing is complete.
+- Refer below step by step guide to create charts and dashboards
+    - [Create Chart](https://dapplooker.notion.site/Create-Charts-9cd44e01cb0f472d835e8f2d954e517a)
+    - [Create Dashboard](https://dapplooker.notion.site/Create-Dashboard-e2023db32c2342969194134a5fb9780b)
+
+
+## Moonriver Popular Dashboards {: #moonriver-explorer }
+
+Track metrics and dashboards of popular Moonriver projects from the Moonriver explorer:
+
+[https://dapplooker.com/category/moonriver?type=dashboard](https://dapplooker.com/category/moonriver?type=dashboard)
+
+![Register your dapp](/images/builders/integrations/analyze/dapplooker/dapplooker-6.png)
+
+## More Resources {: #more-resources }
+
+- **Moonbeam/Moonriver Staking Dashboard with APY data**: [https://network.dapplooker.com/moonbeam/collator](https://network.dapplooker.com/moonbeam/collator)
```

Comment:
this link is just for moonbeam? it might be useful to have a separate link for moonriver